tests/scheduled_checks.py

```diff
@@ -0,0 +1,51 @@
+"""
+These tests are not called on every push or pull request. They're time-intensive, so on GitHub they're only called
+monthly. To run them locally, use the following command:
+
+    pytest -s -rpP tests/scheduled_checks.py
+
+Because the filename doesn't begin with test_, it's not automatically run with pytest. The -s flag is used to print
+the results of the tests, and the -rpP flags are used to suppress the output of the tests that pass.
+
+"""
+from astroquery.simbad import Simbad
+
+from astrodbkit2.utils import _name_formatter
+
+def test_SIMBAD_resolvable(db):
+    # Verify that all sources have valid names
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Add all IDS to the Simbad output as well as the user-provided id
+    Simbad.add_votable_fields("ids")
+    Simbad.add_votable_fields("typed_id")
+
+    simbad_results = Simbad.query_objects(name_list)
+    no_result_rows = []
+    duplicate_rows = []
+    for row in simbad_results[["TYPED_ID", "IDS"]].iterrows():
+        try:
+            name, ids = row[0].decode("utf-8"), row[1].decode("utf-8")
+        except AttributeError:
+            # Catch decoding error
+            name, ids = row[0], row[1]
+
+        simbad_names = [
+            _name_formatter(s)
+            for s in ids.split("|")
+            if _name_formatter(s) != "" and _name_formatter(s) is not None
+        ]
+        if len(simbad_names == 0):
+            no_result_rows += [row]
+
+
+        # Examine DB for each input, displaying results when more than one source matches
+        t = db.search_object(
+            simbad_names, output_table="Sources", fmt="astropy", fuzzy_search=False
+        )
+        if len(t) != 1:
+            duplicate_rows += [row]
```

Comment:
Wow, this is a very thorough test.  This is making sure every possibly SIMBAD identifier also identifies only one database source, right?   This seems extra to me.  It seems like just checking that the first identifier matches only one database source should be enough.

Reply:
I think it currently does the latter? `simbad_results` is only 3 rows for the 3 sources in the database.

Reply:
I don't see how `simbad_results` can be only 3 rows.  A quick search for WASP-76 yielded about a dozen names. The logic here should have that looping over each one.

Reply:
Actually, I see why it's 3 rows now: the DB has 3 rows and that's what Simbad returns back. Each row can have multiple names so for example `simbad_names` can contain `['BD+01 316b', 'WASP-76b']` and `search_object` considers both of them when searching (since it can take a list as an input).